c2corg_ui/templates/sso_login.html

```diff
@@ -0,0 +1,6 @@
+<%inherit file="./base.html"/>
```

Comment:
Since this is is a dummy page, you should probably make sure it is not indexed by adding:
```
<%block name="metarobots"><meta name="robots" content="noindex,follow"></%block>
```
See the default value at https://github.com/c2corg/v6_ui/blob/master/c2corg_ui/templates/base.html#L29

Reply:
Done